### DISCUSSION TITLE
Running blank migration get's stuck for ~60 seconds

### DISCUSSION THREAD

**Discussion by ahhmarr** [mikro-orm/mikro-orm#6230] [General]
### Describe the bug

-  the blank migration command will create the migration file instaneously, but still the commands stays in suspended state and gets finished only after ~60-70 seconds

### Reproduction

1. clone https://github.com/ahhmarr/mikro-orm-stuck-migration/tree/main
2. run `npm install`
3. run `npm run migrate:db:create`

The command should only complete after more than 1 min

### What driver are you using?

@mikro-orm/postgresql

### MikroORM version

6.4.0

### Node.js version

20

### Operating system

Mac Sequoia 15.0.1

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/mikro-orm/mikro-orm/blob/master/CONTRIBUTING.md).
- [X] Read the [docs](https://mikro-orm.io/docs).
- [X] Check that there isn't [already an issue](https://github.com/mikro-orm/mikro-orm/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/mikro-orm/mikro-orm/discussions) or join our [Discord](https://discord.gg/w8bjxFHS7X).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by B4nan** [comment]
Works fine for me, sounds like something in your environment.
